### PR TITLE
Add ruff and update files

### PR DESCRIPTION
**Pull request by emlowe**
Replace black, isort, flake8 and pylint with ruff.
use same `ruff.toml` as `chia-blockchain`
update code as needed.

### CODE REVIEW

File: ruff.toml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    # Its purpose is to prevent accidenatally assigning mutable defaults. However, this is a bit overkill for that.
+    # That being said, it would be nice to add some way to actually guard against that specific mutable default behavior.
+    "RUF009", # function-call-in-dataclass-default-argument
+    # Should probably fix this
+    "RUF029",
+]
+
+
+[lint.flake8-implicit-str-concat]
+# Found 3279 errors.
```
Comment (Quexington) [inline ruff.toml:71]: Not sure this comment is relevant in this repo.

File: tests/cmds/test_inspect.py [hunk 2]
```diff
@@ -187,18 +186,12 @@ def test_spendbundles(self):
         spend_path = Path(__file__).parent.joinpath("object_files/spends/spend.json")
         spend_path_2 = Path(__file__).parent.joinpath("object_files/spends/spend_2.json")
         pubkey: str = "80df54b2a616f5c79baaed254134ae5dfc6e24e2d8e1165b251601ceb67b1886db50aacf946eb20f00adc303e7534dd0"
-        signable_data: str = (
-            "24f5a5fd42d16a20302798ef6ed309979b43003d2320d9f0e8ea9831a92759fb4bccd5bb71183532bff220ba46c268991a3ff07eb358e8255a65c30a2dce0e5fbb"  # noqa
-        )
-        agg_sig: str = (
-            "b83fe374efbc5776735df7cbfb7e27ede5079b41cd282091450e4de21c4b772e254ce906508834b0c2dcd3d58c47a96914c782f0baf8eaff7ece3b070d2035cd878f744deadcd6c6625c1d0a1b418437ee3f25c2df08ffe08bdfe06b8a83b514"  # noqa
-        )
+        signable_data: str = "24f5a5fd42d16a20302798ef6ed309979b43003d2320d9f0e8ea9831a92759fb4bccd5bb71183532bff220ba46c268991a3ff07eb358e8255a65c30a2dce0e5fbb"  # noqa
```
Comment (Quexington) [inline tests/cmds/test_inspect.py:189]: Can these noqa's be disappeared?  I feel like I remember that from the chia-blockchain pass.

**Review by Quexington — CHANGES_REQUESTED** [review]
minor nits.  Also, it would be good in the future to have some mechanism where if we update the ruff.toml in chia-blockchain, it propagates somehow downstream.

**Comment by emlowe** [comment]
Adding `ruff.toml` to the managed files repo might be a good idea - then it would get added to all repos. The downside is making changes to it is more complicated

File: tests/cmds/test_inspect.py (hunk 2 above)
Comment (emlowe) [reply tests/cmds/test_inspect.py:189]: The long strings are kinda annoying to disappear - eg, using `()`, triple quotes, `\`, etc

File: ruff.toml (hunk 1 above)
Comment (emlowe) [reply ruff.toml:71]: It's not, but I'm trying to keep the file the same

File: tests/cmds/test_inspect.py (hunk 2 above)
Comment (emlowe) [reply tests/cmds/test_inspect.py:189]: It would be better to be specific I suppose to specifically ignore long lines rather than blanket noqa

Comment (emlowe) [reply tests/cmds/test_inspect.py]: cleaned up using ()

**Review by Quexington — APPROVED** [review]

**Pull request merged**